src/Impostor.Api.Innersloth.Generator/Impostor.Api.Innersloth.Generator.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <DebugType>embedded</DebugType>
+    <Nullable>enable</Nullable>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.7.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
+    <PackageReference Include="PolySharp" Version="1.13.2" PrivateAssets="all" />
+
+    <PackageReference Include="System.Text.Json" Version="7.0.3" Private="true" GeneratePathProperty="true" />
+    <PackageReference Include="CSharpPoet" Version="0.3.0" Private="true" GeneratePathProperty="true" />
```

Comment:
Nice library, did you write it just for this?

Reply:
Actually I [kinda did](https://github.com/js6pak/Impostor/blob/bfbd01256db37e64d08da0b75aed304c487de574/src/Impostor.Generator/Impostor.Generator.csproj#L20) 2 years ago but then used it in other projects before finishing this Impostor change.